02_configure_host.sh

```diff
@@ -153,15 +163,15 @@ if [ "${RHEL8}" = "True" ] ; then
     sudo firewall-cmd --zone=libvirt --add-port=6230-${VBMC_MAX_PORT}/udp
     sudo firewall-cmd --zone=libvirt --add-port=6230-${VBMC_MAX_PORT}/udp --permanent
 else
-    if ! sudo iptables -C INPUT -i baremetal -p udp -m udp --dport 6230:${VBMC_MAX_PORT} -j ACCEPT 2>/dev/null ; then
-        sudo iptables -I INPUT -i baremetal -p udp -m udp --dport 6230:${VBMC_MAX_PORT} -j ACCEPT
+    if ! sudo $IPTABLES -C INPUT -i baremetal -p udp -m udp --dport 6230:6235 -j ACCEPT 2>/dev/null ; then
+        sudo $IPTABLES -I INPUT -i baremetal -p udp -m udp --dport 6230:6235 -j ACCEPT
```

Comment:
Sorry just noticed this: can you put back VBMC_MAX_PORT?

Otherwise this is good to go.

Reply:
yup